finetuner/__init__.py

```diff
@@ -296,3 +307,82 @@ def get_token() -> str:
     :return: user token as string object.
     """
     return ft.get_token()
+
+
+def get_model(
```

Comment:
I dont like the get_model name, maybe `build_engine`?

Reply:
no, think about when we support pytorch, it's not engine anymore